src/test/java/com/solvd/micro9/vkontaktah/service/TestDataProvider.java

```diff
@@ -0,0 +1,183 @@
+package com.solvd.micro9.vkontaktah.service;
+
+import com.solvd.micro9.vkontaktah.domain.Like;
+import com.solvd.micro9.vkontaktah.domain.Post;
+import com.solvd.micro9.vkontaktah.domain.User;
+import org.junit.jupiter.params.provider.Arguments;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+public final class TestDataProvider {
```

Comment:
@eyakauleva I think creation of objects should be visible in your tests

Reply:
@h1alexbel that's why I decided to make parameterized tests so it's immediately visible from where test data came from

Reply:
@eyakauleva test must be simple, not long or short
so, if data is visible in your tests is better, than chasing the short-lined test

let's assume that someone tomorrow will change the assertions no matter of what
tests will be failed, and it requires time to dig it and find where is the source of data that comes to us

WDYT?

Reply:
@h1alexbel but when using, for example, fake objects people also have to check data in another classes

Reply:
@eyakauleva fake objects are not data holders
fake objects are needed mainly for mocking

your tests require just objects that you have now